source/grammar/qasm3.g4

```diff
@@ -237,8 +237,8 @@ quantumMeasurement
     ;
```

Comment:
The line above `quantumMeasurement`, should only take `indexIdentifier`. That is,
```
quantumMeasurement    
    : 'measure' indexIdentifier
    ;
```

Reply:
@1ucian0 pls update this. We do not allow for measuring multiple individual qubits at once. In order to measure multiple qubits in the same instruction, you'd need to use a qubit register. See https://qiskit.github.io/openqasm/language/insts.html#measurement

Reply:
done in 21ee0533ec3ac06bf93d969885ed34ea6e77306b